ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -270,7 +242,7 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
     }, 300);
   }
 
-  updateProfile(attribute: string, value: string) {
+  updateProfileToBeDeleted(attribute: string, value: string) {
```

Comment:
What does this name mean?

Reply:
This method will be deleted once we have the feature flag is set to true for PROD.